lib/valkyrie/indexing/solr/indexing_adapter.rb

```diff
@@ -21,8 +22,35 @@ def initialize(connection: default_connection, resource_indexer: default_indexer
           @resource_indexer = resource_indexer
         end
 
+        def save(resource:)
+          persist([resource])
+        end
+
         private
 
+          def persist(resources)
+            documents = resources.map do |resource|
+              solr_document(resource)
+            end
+            add_documents(documents)
+          end
+
+          def solr_document(resource)
+            base_hash(resource).merge(resource_indexer.new(resource: resource).to_solr)
```

Comment:
would it be good to just push the base hash info into the indexer as well?

Reply:
That's a good idea once we have a base indexer. currently the only indexer is the Null indexer I'm using from Valkyrie. Would you like me to add a base indexer in this PR or migrate this in a later PR when that's introduced?

Reply:
later is fine. approving.